pkg/probes/legacy/machine_images.go

```diff
@@ -1,17 +1,17 @@
 package legacy
 
 import (
+	platform "github.com/openshift/osd-network-verifier/pkg/data/cloud"
 	"github.com/openshift/osd-network-verifier/pkg/data/cpu"
-	"github.com/openshift/osd-network-verifier/pkg/helpers"
 )
 
 // cloudMachineImageMap is a lookup table mapping VM image IDs to their
 // respective cloud platforms, CPU architectures, and cloud regions. To
 // access, reference cloudMachineImageMap[$CLOUD_PLATFORM][$CPU_ARCH][$REGION];
-// e.g., cloudMachineImageMap[helpers.PlatformAWS][cpu.ArchX86]["us-east-1"]
+// e.g., cloudMachineImageMap[platform.AWSClassic][cpu.ArchX86]["us-east-1"]
 // Note that the legacy probe only has ever supported X86 on AWS
 var cloudMachineImageMap = map[string]map[cpu.Architecture]map[string]string{
-	helpers.PlatformAWS: {
+	platform.AWSClassic.String(): {
```

Comment:
Because the `Platform` type is comparable, you can use it as a map key. IOW, the line above should look like
```go
var cloudMachineImageMap = map[platform.Platform]map[cpu.Architecture]map[string]string{
```
And then you can use `platform.AWSClassic` directly w/o the `.String()`